proto/client/gClient.go

```diff
@@ -50,19 +50,22 @@ type ConfigClient struct {
 }
 
 type ConfClient interface {
-	// channel is created on which subscription is done.
+	// A channel is created on which subscription is done.
 	// On Receiving Configuration from ConfigServer, this api publishes
 	// on created channel and returns the channel
-	PublishOnConfigChange(bool) chan *protos.NetworkSliceResponse
+	PublishOnConfigChange(metadataRequested bool, stream protos.ConfigService_NetworkSliceSubscribeClient) chan *protos.NetworkSliceResponse
```

Comment:
We have 2 set of NFs

1. One set of NFs use PublishOnConfigChange. SMF, AMF, PCF falls in this category 
2. Other set of NFs use subscribeToConfigPod - UDR, ausf, nssf,udm, nrf 

We should try to see if we can bring consistency here.

Reply:
I suggest to use subscribeToConfigPod in all NF's.

Reply:
I am planning to open PR's in each NF to unify the implementation as described above.

Reply:
Thank you. It makes sense. I shall review the code.

Reply:
Thank you!